src/entities/campaign/components/CampaignSettings.tsx

```
@@ -161,7 +161,18 @@ export const CampaignSettings: React.FC<CampaignSettingsProps> = ({ campaignId }
         <div className="mt-8 w-full rounded-[12px] border border-solid border-[#DBDBDB] p-6">
           <div>
             <h1 className="mb-4 text-xl font-semibold">{campaign?.name}</h1>
-            <p className="text-[#292929]">{campaign?.description}</p>
+            <div
+              className="prose prose-sm max-w-none"
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)